ServerCore/Pages/Player/Create.cshtml

```diff
@@ -43,16 +43,66 @@ else
 
             @if (Model.Event.HasIndividualLunch)
             {
-                    <div class="form-group">
-                        <label asp-for="PlayerInEvent.Lunch" class="control-label"></label>
-                        <input asp-for="PlayerInEvent.Lunch" class="form-control" />
-                        <span asp-validation-for="PlayerInEvent.Lunch" class="text-danger"></span>
-                    </div>
-                    <div class="form-group">
-                        <label asp-for="PlayerInEvent.LunchModifications" class="control-label"></label>
-                        <input asp-for="PlayerInEvent.LunchModifications" class="form-control" />
-                        <span asp-validation-for="PlayerInEvent.LunchModifications" class="text-danger"></span>
-                    </div>
+                <div class="form-group">
+                    <label asp-for="PlayerInEvent.Lunch" class="control-label"></label><br />
+                    @{
```

Comment:
Future: There should be a common component that is used on both pages here. I'm not super familiar with how to do that, so if you choose not to at least put a big comment at the top of this block and the one in Edit.cshtml so they don't diverge.

Reply:
Also, am I correct that we won't be exercising this code at all for PD2024 since lunch is a team lunch?

Reply:
I was thinking the same thing since this code, the one on the player edit page, and the swag registration page are identical, so as long as I'm making changes, I'd might as well do this as well.  And while yes, we're not using it this year, the fact that this was dead code in the codebase before this change was something that needed addressed anyway.

Reply:
Oh wait, partials apparently can't save data according to https://learn.microsoft.com/en-us/aspnet/core/mvc/views/partial?view=aspnetcore-7.0, so this is indeed just going to be relegated to the future.  Comment it is!